fleet_vehicle_notebook/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module extends the functionality of fleet management. It allows the registration of a vehicle's fuel capacity
```

Comment:
Please update the description, also mentioning that this module doesn't really do anything by itself. 
it's just mean to be used by other modules

Reply:
done